src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -58,17 +74,33 @@ function DiscountRateExtra({ hasDuration }: { hasDuration?: boolean }) {
           </Trans>
         </FormItemWarningText>
       )}
-      <Trans>
-        The ratio of tokens rewarded per payment amount will decrease by this
-        percentage with each new funding cycle. A higher discount rate will
-        incentivize supporters to pay your project earlier than later.
-      </Trans>
+      <p>
+        <Trans>
+          The issuance rate will decrease by this percentage with each new
+          funding cycle. A higher discount rate will incentivize supporters to
+          pay your project earlier than later.
+        </Trans>
+      </p>
+      {discountRatePercent > 0 && (
+        <>
+          <br />
```

Comment:
**suggestion** best to avoid `br` because it can lead to unintended consequences is some styling changes somewhere that affects it.

Chucking a margin on TabDescription (or a wrapping div) should be fine.